fixed an issue with https for both public and private repositories (#2666)
buildkite/docs: pages/pipelines/hosted_agents/pipeline_migration.md

- Each step in the pipeline targets the required hosted agent queue.
- Source control settings have been updated to allow code access.

> An additional step is required for private private repositories, see below for the relevant instructions.

Reviewer: ```suggestion
An additional process is required for private private repositories, see below for the relevant instructions.
```

1. Select the repository.
1. Select _Save Repository_.

> ## All repositories

Reviewer: If this is 'all repositories', doesn't this include private repositories too?

> When accessing a public repository from a Buildkite hosted agent, you must ensure the services is using `HTTPS` for checkout.

Reviewer: ```suggestion
When accessing any repository (public or private) from a Buildkite hosted agent, you must also ensure the repository is checked out using `HTTPS`.
```
